### PR TITLE
RD2: Metadata keeps Day of Month values of 29 and 30 as active

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
As an Admin of an Org with Enhanced Recurring Donations enabled, I want to be able to assign a Day of Month of 29 or 30 to a Recurring Donation record.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
**lurch: attach W-040426

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001Hu46QAC" target="_blank">W-040426</a>

**Comment by andrewyu-salesforce** [comment]
@RadSFDeveloper The change is ready for review.

File: unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object [hunk 1]
```diff
@@ -94,7 +94,7 @@
     <fields>
         <fullName>%%%NAMESPACE%%%Day_of_Month__c</fullName>
         <externalId>false</externalId>
-        <inlineHelpText>Sets the specific day of the month for future installment Opportunities when the Installment Period is Monthly.</inlineHelpText>
+        <inlineHelpText>Sets the specific day of the month for future installment Opportunities when the Installment Period is Monthly. If you select 29 or 30, the installment date will be the last day of the month for months that don&apos;t have that many days.</inlineHelpText>
```
Comment (andrewyu-salesforce) [inline unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object]: @sam-knox Please review the updated help text.

**Pull request merged**